Sources/TransferMethod/TransferMethodTypeDetail.swift

```diff
@@ -24,6 +24,14 @@ struct TransferMethodTypeDetail {
     let transferMethodType: String
     var fees: [HyperwalletFee]?
     var processingTime: String?
+    var currency: String
+
+    init(_ currency: String, _ transferMethodType: HyperwalletTransferMethodType) {
```

Comment:
Will add currency in HyperwalletFee so no need to pass it separately

Reply:
Delete this file and use HyperwalletTransferMethodType extension instead

Reply:
fixed